pmm-app/src/pmm-inventory/ColumnRenderers/ColumnRenderers.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { MAIN_COLUMN } from 'pmm-inventory/Inventory.constants';
+import { CustomLabel } from 'pmm-inventory/Inventory.types';
+
+export const servicesDetailsRender = element => {
+  const labels = Object.keys(element).filter(label => !MAIN_COLUMN.includes(label));
+  return (
+    <div className="other-details-wrapper">
+      {labels.map((label, accessor) =>
+        element[label] ? <span key={accessor}>{`${label}: ${element[label]}`}</span> : null
```

Comment:
Why not just
```suggestion
        element[label] && <span key={accessor}>{`${label}: ${element[label]}`}</span>
```
?

Reply:
Both are OK, let's leave this choice to the developer )